### DISCUSSION TITLE
Add site for squidfunk.github.io

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#4878] [General]
This isn't really something related to mkdocs-material but I couldn't think of a better place to suggest this, so here we go.

You, Squidfunk, should consider making a site accessible under the squidfunk.github.io domain, as right now, it leads to a 404 page.
This is kinda annoying as browsers like Firefox have the bad behaviour of only suggesting auto-completions for the main domain, before giving suggestions for subdomains (i.e. typing `squ` suggests `squidfunk.github.io` but not `squidfunk.github.io/mkdocs-material` until I complete the domain and add a forward slash to it).

It's a minor inconvenience, but it would help and could perhaps even be a good place for you to promote your work and experience.

**Comment by squidfunk** [comment]
Thanks for suggesting. It's the very reason I hate Firefox haha. I'll see if I can find some time to deploy a site.
